Classes/Command/CacheCommandController.php

```diff
@@ -122,4 +129,25 @@ public function listGroupsCommand()
                 break;
         }
     }
+
+    /**
+     * Flush cache for $cacheType with data handler implementation
+     *
+     * @param string $cacheType either "system" or "all"
+     * @return void
+     * @throws \UnexpectedValueException
+     */
+    public function flushCacheWithDataHandlerHandlerCommand($cacheType)
```

Comment:
I would like to see this functionality rather in the cache service.

But there are some conceptual questions to be answered beforehand.

* Which hook do you have in mind to be cleared? `clearPageCacheEval` or `clearCachePostProc`?
* From a user perspective, what is the difference between `cache:flushcachewithdatahandler system` and `cache:flushgroups system`?
* Can't we just call the hooks (with a fake DataHandler instance) when we flush "system" or "all" cache?
* What should we do with the `clearPageCacheEval` hook?